[IMP] Employees - Certifications (#16307)
odoo/documentation: content/applications/hr/employees/certifications.rst

Certifications
==============

When jobs require specific knowledge, training, or certifications, it is necessary to track employee
certifications to ensure the employee is properly trained, and the necessary documentation is in

> place.

Reviewer: ```suggestion
When jobs require specific knowledge, training, or certifications, it is necessary to track employees'
certifications to ensure they are properly trained and that the necessary documentation is in place.
```
Revised the sentence

Configuration --> Skill Types`. Click the :guilabel:`New` button in the upper-left corner, and a
blank *Skill Types* form loads.

Enter the following information on the form:

- :guilabel:`Skill Type`: Enter the type of certification being added. This acts as the *category*,
  and individual certifications are nested beneath it.
- :guilabel:`Color`: Click on the existing color to view the available colors. Click the desired
  color to select it.
- :guilabel:`Certification`: Click the toggle to indicate the skill is a certification. The toggle
  turns green when it is active.
- :guilabel:`Skills`: Click :guilabel:`Add a line` and enter the :guilabel:`Name` for the specific
  certification.
- :guilabel:`Levels`: Click :guilabel:`Add a line`, and enter a :guilabel:`Name` and
  :guilabel:`Progress` percentage (`0`-`100`) for each level. Set a :guilabel:`Default Level` by
  clicking the toggle on the desired line (only one level can be selected).

.. example::
   A United States-based company requires its employees to go through various :abbr:`OSHA
   (Occupational Safety and Health Administration)` training, and requires certification that the

>    training is complete.

Reviewer: ```suggestion
   A United States-based company requires its employees to complete various :abbr:`OSHA
   (Occupational Safety and Health Administration)` training and to certify that the training is complete.
```
Revised sentence

>    The company creates a new skill type, and enters :guilabel:`OSHA Training` in the

Reviewer: ```suggestion
   The company creates a new skill type and enters :guilabel:`OSHA Training` in the
```
Removed comma

determine which employees need to keep necessary certifications current in the default list view. In
this scenario, it is beneficial to view the certifications by expiration status.

To do so, navigate to :menuselection:`Employees app --> Learning --> Certifications`. Next, clear
the default :icon:`oi-group` :guilabel:`Type` grouping in the search bar. Next, click the
:icon:`fa-caret-down` :guilabel:`(Toggle Search Panel)` icon, then click :guilabel:`Custom Group`
:icon:`fa-caret-down`, revealing a drop-down menu. Click :guilabel:`Validity Stop`, then click away
from the drop-down menu to close it.

After doing so, all the certifications are grouped by the expiration month, in descending

> chronological order, with the oldest expiration month at the top.

Reviewer: ```suggestion
After doing so, all the certifications are grouped by expiration month, in descending
 order, with the oldest  at the top.
```
Revised sentence

The entries are color-coded. Current certifications that are still valid appear in black, expired
certifications appear in gray, and certifications that are going to expire within the next 90 days
appear in orange.

.. image:: certifications/status.png
   :alt: The list of employee certifications, grouped by status.

.. tip::
   The time frame is changeable to either :guilabel:`Year`, :guilabel:`Quarter`, :guilabel:`Week`,
   or :guilabel:`Day`. To change the presented timeframe groups, click into the search bar, then
   click :icon:`fa-check` :guilabel:`Validity Stop` :icon:`fa-caret-down`, and click the desired

>    time frame.

Reviewer: ```suggestion
   or :guilabel:`Day`. To change the presented timeframe groups, click the search bar, then
   click :icon:`fa-check` :guilabel:`Validity Stop` :icon:`fa-caret-down`, and select the desired
   time frame.
```
Revised sentence

> Certifications`. Click :guilabel:`New`, and a blank certification form loads. Enter the following

Reviewer: ```suggestion
Certifications`. Click :guilabel:`New`, to load a blank certification form. Enter the following
```
Revised sentence

information on the form:

- :guilabel:`Employee`: Select the employee who received the certification using the drop-down menu.
- :guilabel:`Category`: Click the *type* of certification received.
- :guilabel:`Skill`: Select the specific certification received. The presented options change if the
  :guilabel:`Category` is changed.
- :guilabel:`Skill Level`: If the selected certification has a set of *skill levels* associated with

>   it, the levels appear in this section. Click on a level to select the level achieved from the

Reviewer: ```suggestion
- :guilabel:`Skill Level`: If the selected certification has a set of *skill levels*, those levels appear in this section. Click on a level to select the level achieved from the
```
Revised sentence

  certification.
- :guilabel:`Validity`: Set the validity start and end dates in the two fields. The current date

>   populates the first field, and :guilabel:`indefinite` populates the :guilabel:`To` field, by

Reviewer: ```suggestion
- :guilabel:`Validity`: Set the validity start and end dates in the two fields. The current date
  is populated in the first field, and :guilabel:`indefinite` is populated in the :guilabel:`To` field by
```
Revised sentence

  default.

Once all the fields are configured, click the :guilabel:`Save` button. The certification is logged
for the employee, and appears on the *Certifications* report, and the employee record.

.. image:: certifications/add-cert.png
   :alt: A certification form filled out for a 10-hour OSHA safety course.

Certifications Report
=====================

> To view a report of all employee certifications, navigate to :menuselection:`Employees app -->

Reviewer: ```suggestion
To view a report of all employee certifications, navigate to the :menuselection:`Employees app -->
```
Added the
